napari/_qt/_qapp_model/_tests/test_layerlist_context_actions.py

```diff
@@ -27,6 +27,11 @@ def test_layer_actions_ctx_menu_execute_command(
     if command_id == 'napari.layer.merge_stack':
         with pytest.raises(IndexError, match=r'images list is empty'):
             app.commands.execute_command(command_id)
+    elif command_id == 'napari.layer.merge_rgb':
+        with pytest.raises(
+            AssertionError, match='Merging to RGB requires exactly 3 Image' ''
```

Comment:
```suggestion
            AssertionError, match='Merging to RGB requires exactly 3 Image' ''
```

Question, why the `' "` at the end?

Reply:
uh 👀  
copy pasta? 😬